<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFT Spectrogram</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="displayTab">
         <attribute name="title">
          <string>Display</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <spacer name="leftSpacerStartStopButton">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="startStopButton">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="rightSpacerStartStopButton">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="WaterfallWidget" name="waterfallWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settingsTab">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="0">
             <widget class="QLabel" name="sampleRateLabel">
              <property name="text">
               <string>Frequency (Hz)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="intervalSpinBox">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="intervalLabel">
              <property name="text">
               <string>Interval (ms)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="sampleRateComboBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>8000</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11250</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>22500</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>44100</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>48000</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="bufferSizeLabel">
              <property name="text">
               <string>Buffer Size</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="bufferSizeSpinBox">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>400</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WaterfallWidget</class>
   <extends>QWidget</extends>
   <header>waterfallwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
